resources/lang/en/groups.php

```diff
@@ -98,13 +99,17 @@
   'volunteers_nearby' => 'Volunteers nearby',
   'share_group_stats' => 'Share group stats',
   'about' => 'About',
-  'read_more' => 'Read more',
+  'about_none' => 'There is no description for this group yet.',
+  'read_more' => 'READ MORE&nbsp;<img class="icon" src="/images/add-icon.svg" style="width: 20px;" />',
+  'read_less' => '<img class="icon" src="/images/minus-icon.svg" style="width: 20px;" />&nbsp;READ LESS',
   'volunteers' => 'Volunteers',
   'invite_to_group' => 'Invite to group',
   'key_stats' => 'Key stats',
-  'participants' => 'Participants',
-  'hours_volunteered' => 'Hours volunteered',
-  'total_events' => 'Total events',
+  'group_facts' => 'Group facts',
```

Comment:
It's what the design says, but "facts" is a dull word.  "Achievements"?

Reply:
Agreed!  I'll change it to achievements.